projects/online/online/cli.py

```diff
@@ -31,19 +33,61 @@ def build_parser():
     return parser
 
 
-def cli(args=None):
-    parser = build_parser()
-    args = parser.parse_args(args)
-    # Create a new log file each time we start using the current UTC time
-    logdir = args.outdir / "log"
-    logdir.mkdir(exist_ok=True, parents=True)
-    log_suffix = datetime.now(tz=timezone.utc).strftime("%Y-%m-%dT%H:%M:%S")
-    configure_logging(
-        args.outdir / "log" / f"deploy_{log_suffix}.log", args.verbose
+def configure_logging(logdir: Path, verbose: bool = False):
+    log_format = "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+    logging.basicConfig(
+        format=log_format,
+        level=logging.DEBUG if verbose else logging.INFO,
+        stream=sys.stdout,
+    )
+
+    logger = logging.getLogger()
+
+    # set logging to use UTC time
+    logging.Formatter.converter = lambda *args: datetime.now(
+        tz=timezone.utc
+    ).timetuple()
+
+    timestamp = datetime.now(tz=timezone.utc).strftime("%Y-%m-%dT%H:%M:%S")
+    run_log_dir = logdir / timestamp
+    run_log_dir.mkdir(exist_ok=True, parents=True)
+
+    # set up the timed rotating file handler
+    formatter = logging.Formatter(log_format)
+
+    # ensure formatter also uses UTC time
+    formatter.converter = lambda *args: datetime.now(
+        tz=timezone.utc
+    ).timetuple()
```

Comment:
What's the difference between this and the `logging.Formatter.converter = ...` line above?

Reply:
I think they're both necessary but not sure why